printer_zpl2/wizard/wizard_import_zpl2.py

```diff
@@ -345,7 +345,9 @@ class WizardImportZPl2(models.TransientModel):
     _description = "Import ZPL2"
 
     label_id = fields.Many2one(
-        comodel_name="printing.label.zpl2", string="Label", required=True, readonly=True
+        comodel_name="printing.label.zpl2",
+        string="Label",
+        required=True,
```

Comment:
I wouldn't remove readonly here

Reply:
Agreed, it doesn't make sense to move it to the view definition within a wizard